integration_testing/features/android/change-facility-after-device-setup.feature

```diff
@@ -0,0 +1,146 @@
+Feature: Change facility after device setup
+
+	Background:
+    Given I’ve set up my Kolibri device with the *On my own* device setup option
+			And I am signed in
+			And I am at the *Profile* page
+
+	Scenario: Change learning facility - default path
+		When I click the *Change* button
+		Then I see *Select* facility
+			And I see a list with facilities
+			And I see that the first facility in the list is selected
+			And I see an *Add new address* link
+		When I click *Continue*
+		Then I see a *Change facility* label
+			And I see text that I am about to move my account and progress data to the selected ‘<facility>’
```

Comment:
There's also one conditional line of text that appears for accounts with non-learner roles - "Your user type will change from ‘[non-learner-role] to ‘learner’. You will need an admin to make you a(n) ‘[non-learner-role]’ again."